vcr/revocation/statuslist2021_issuer.go

```diff
@@ -77,8 +77,8 @@ type credentialRecord struct {
 	SubjectID string `gorm:"primaryKey"`
 	// StatusPurpose is the purpose listed in the StatusList2021Credential.credentialSubject
 	StatusPurpose string
-	// Expanded StatusList2021 bitstring
```

Comment:
this should stay Expanded? It is accessed in go as the expanded bitstring

Reply:
```
`gorm:"column:bitstring"`
```
links the field to the column

Reply:
It's become `bitstring` as you suggested, which is fine because the SQL marshallers hande how to store is (it compresses/expand it).

Reply:
And that is all fine, but during usage you want to know if it is the expanded or compressed form so I know what to do with it. Since the value in the struct is always in its expanded form I think the name should not change.

Reply:
updated comment probably also works